test_data/test_semantic_parser/source/page3.txt

```diff
@@ -0,0 +1,11 @@
+  
+.. _connection-limits:
+
+===========
+A heading!
```

Comment:
Is this test case basically the same as that in `index.txt`?

Reply:
Oops it is  👀